content/en/docs/languages/python/instrumentation.md

```diff
@@ -396,7 +396,37 @@ meter.create_observable_gauge(
 
 ## Logs
 
-The logs API & SDK are currently under development.
+The logs API & SDK are currently under development. To start collecting logs, you need to initialize a
```

Comment:
Thanks @hectorhdzg I think this is good to have back.

@open-telemetry/docs-approvers , is this paragraph cohesive/consistent with similar pages? Else is something like a warning box used for other non-stable features?

Reply:
Yep, this is a fine enough disclaimer.